src/Microsoft.AspNetCore.OData/Formatter/ODataOutputFormatter.cs

```diff
@@ -225,7 +219,7 @@ public override Task WriteResponseBodyAsync(OutputFormatterWriteContext context,
 
                 ODataSerializerProvider serializerProvider = request.GetRequestContainer().GetRequiredService<ODataSerializerProvider>();
 
-                ODataOutputFormatterHelper.WriteToStream(
+                return ODataOutputFormatterHelper.WriteToStreamAsync(
```

Comment:
shall we call ".ConfigureAwait(false);" after await?

Reply:
Not required for ASP.NET Core components.

Reply:
@ - This file is only used in .NET Core, but some of the shared files are used by both .NET Core and .NET Classic.

For the shared files, do we need the .ConfigureAwait(false) everywhere we use await in order to avoid possible deadlocks?  If so, should we use an #IF !NETCORE to add the .ConfigureAwait(false), or should we just always add it to the common calls (i.e., is there any disadvantage to adding in .NET Core)?

---
In reply to: [515953430](https://github.com/OData/WebApi/pull/2336#discussion_r515953430) [](ancestors = 515953430)

Reply:
Reverted async logic for .NET Classic. It's not needed there, and introduces unnecessary churn.

---
In reply to: [520190988](https://github.com/OData/WebApi/pull/2336#discussion_r520190988) [](ancestors = 520190988,515953430)